### ISSUE TITLE
proc stat never emitting with my nexus 5

### ISSUE THREAD

**Issue by dweinstein** [CyberAgent/adbkit#21]
I'm playing with a nexus 5 device and trying out the `client.openProcStat` code:

```
      return client.openProcStat(device.id)
      .then(function(stat) {
        stat.on('load', function (data) {
          debugger;
        });
        stat.on('error', function (err) {
          console.log(err);
          debugger;
        });
      });
```

For me the `load` event is never emitted, and after setting some debug statements it seems the `ProcStat.prototype.parse` is never called on (`update` is though)

I know that other things are working, like the [checking for nfc](https://github.com/CyberAgent/adbkit#checking-for-nfc-support) example works fine.

I feel like this ins't enough info to go on, so let me know what else might help with debugging.

**Comment by sorccu** [comment]
Looks like the whole implementation was broken due to previous API changes. While we do have many tests this is one of the methods they do not currently cover.. :(

Released 2.1.6 with a fix.

**Comment by dweinstein** [comment]
Awesome! I'll be sure to check it out! Thanks!
